### ISSUE TITLE
Disable rule for multi-line statement

### ISSUE THREAD

**Issue by tuukkamustonen** [PyCQA/pylint#3429]
Having something like:

```python
def func(
    foo
):
    pass
```

Pylint gives:

```
foo.py:1:0: C0102: Black listed name "foo" (blacklisted-name)
...
```

This cannot be disabled like:

```python
def func(  # pylint: disable=blacklisted-name
    foo  # pylint: disable=blacklisted-name
):  # pylint: disable=blacklisted-name
    pass
```

None of these work. A [SO answer](https://stackoverflow.com/a/25779494/165629) suggests to:

```python
# pylint: disable=blacklisted-name
def func(
    foo
):
    # pylint: enable=blacklisted-name
    pass
```

But that's pretty annoying...

I would prefer if this was possible:

```python
def func(
    foo
):  # pylint: disable=blacklisted-name
    pass
```

**Comment by rahulrachh** [comment]
are you running pylint from cmd ? if yes then **instead of** "pylint filename.py" go for "pylint --disable=C0102 filename.py" **OR** "pylint --disable=blacklisted-name filename.py" This will disable the warning that you are seeing

**Comment by PCManticore** [comment]
I'm sorry, I cannot reproduce this with latest version installed from `master` on Python 3.7.X:
```
a.py:1:0: I0011: Locally disabling blacklisted-name (C0102) (locally-disabled)
a.py:2:0: I0011: Locally disabling blacklisted-name (C0102) (locally-disabled)
a.py:3:0: I0011: Locally disabling blacklisted-name (C0102) (locally-disabled)
a.py:2:0: C0330: Wrong hanging indentation before block (add 4 spaces).
    foo  # pylint: disable=blacklisted-name
    ^   | (bad-continuation)
a.py:1:0: C0114: Missing module docstring (missing-module-docstring)
a.py:1:0: C0116: Missing function or method docstring (missing-function-docstring)
a.py:2:4: W0613: Unused argument 'foo' (unused-argument)
a.py:2:0: I0021: Useless suppression of 'blacklisted-name' (useless-suppression)
a.py:3:0: I0021: Useless suppression of 'blacklisted-name' (useless-suppression)
a.py:1:0: I0020: Suppressed 'blacklisted-name' (from line 1) (suppressed-message)
```

Also the pragma controls for disabling messages are somewhat finicky as they depend on the line numbers of the node you intend to disable the error for. This means for expressions spanning on multiple lines, such as this function's signature, the pragma needs to be placed at a position that can contain the desired element, and in this case, above the function should do the trick. 

I'm afraid we won't be changing this anytime soon.

**Comment by tuukkamustonen** [comment]
Thanks for the quick response!

It's interesting that the output from master is dramatically different. Mine is just:

```
************* Module foo
foo.py:2:0: C0330: Wrong hanging indentation before block (add 4 spaces).
    foo  # pylint: disable=blacklisted-name
    ^   | (bad-continuation)
foo.py:1:0: C0102: Black listed name "foo" (blacklisted-name)
foo.py:1:0: C0114: Missing module docstring (missing-module-docstring)
foo.py:1:0: C0116: Missing function or method docstring (missing-function-docstring)
foo.py:2:4: W0613: Unused argument 'foo' (unused-argument)
```

I'm running (sorry, forgot to mention this in original report):

```
pylint 2.4.4
astroid 2.3.1
Python 3.5.5 (default, May 29 2018, 10:15:24) 
[GCC 5.4.0 20160609]
```

I assume some improvements have been made in master (`useless-suppression`), neat.

> Also the pragma controls for disabling messages are somewhat finicky as they depend on the line numbers of the node you intend to disable the error for. This means for expressions spanning on multiple lines, such as this function's signature, the pragma needs to be placed at a position that can contain the desired element, and in this case, above the function should do the trick.

@PCManticore Yeah, I understand that it's technically difficult... However, having `# pylint: disable=...` _above_ requires `# pylint: enable=...` below which introduces bloat. Could we have some syntax that would disable the rule only for the _next statement_? Like `# pylint: disable-stm=...` or similar, as a workaround?
